src/order/bounded_order.lean

```diff
@@ -881,6 +881,44 @@ option.rec h₁ h₂
 @[simp] lemma rec_top_coe_coe {C : with_top α → Sort*} (d : C ⊤) (f : Π (a : α), C a)
   (x : α) : @rec_top_coe _ C d f ↑x = f x := rfl
 
+/-- `with_top.to_dual` is the equivalence sending `⊤` to `⊥` and any `a : α` to `to_dual a : αᵒᵈ`.
+See `with_top.to_dual_bot_equiv` for the related order-iso.
+-/
+protected def to_dual : with_top α ≃ with_bot αᵒᵈ := to_dual
```

Comment:
To me this is more confusing than
```suggestion
protected def to_dual : with_top α ≃ with_bot αᵒᵈ := equiv.refl _
```
Can you explain what the intuition of your spelling is?

Reply:
I think neither spelling captures the intent properly. My intuition was that I want to send the alpha to alpha-dual, so that's why I am using `to_dual`, which is usually `α ≃ αᵒᵈ`. The larger goal is to not prove lemmas about `with_top _` using `with_bot _`  (or vice versa) by API-penetrating dual lemma, but to use rewrites.

Reply:
The definition doesn't matter so long as you provide the correct `_apply` lemmas. Nevertheless, I understand Eric's spelling better than yours.